desktop/mac/release-notes/index.md

```diff
@@ -33,6 +33,32 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
 > chip](https://desktop.docker.com/mac/main/arm64/Docker.dmg?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-mac-arm64){:
 > .button .primary-btn }
 
+### New
+
+- Add Desktop Version to 'docker version' output.
+
+### Upgrades
+
+- Bump ECR Credential helper to 0.6.0
+
+### Bug fixes and minor changes
+
+- Fixed VM startup hang when using `osxfs` and no host directories are shared with the VM.
+- Increased filesystem watch ("inotify") limits by setting `fs.inotify.max_user_watches=1048576` and 'fs.inotify.max_user_instances=8192' in Linux.
+- Fixed an issue related to compose app started with version 2, but the dashboard only deals with version 1.
+- Fixed an issue related to Docker Desktop login lost after restart.
+- Fixed issues related to the About application menu.
```

Comment:
Unable to open the About Docker Desktop window. 

@mat007 Could you pls check if it affects both Mac and Windows.